integration_tests/test_suites/backcompat-test-suite/tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist=py{38,37,36}-{unix,windows}-{dagit-latest-release,dagit-earliest-release,user-code-latest-release,user-code-earliest-release},pylint
+skipsdist = True
+
+[testenv]
+passenv = CI_* COVERALLS_REPO_TOKEN BUILDKITE BACKCOMPAT_TESTS_DAGIT_HOST DAGSTER_EARLIEST_RELEASE
+
+deps =
+  -e ../../../python_modules/dagster[test]
+  -e ../../../python_modules/dagster-graphql
+  -e ../../../python_modules/dagster-test
+  -e ../../../python_modules/libraries/dagster-postgres
+
+whitelist_externals =
+  /bin/bash
+  echo
+commands =
+  echo -e "--- \033[0;32m:pytest: Running tox tests\033[0m"
+  dagit-latest-release: pytest -m "dagit-latest-release" -vv -s {posargs}
+  dagit-earliest-release: pytest -m "dagit-earliest-release" -vv -s {posargs}
+  user-code-latest-release: pytest -m "user-code-latest-release" -vv -s {posargs}
+  user-code-earliest-release: pytest -m "user-code-earliest-release" -vv -s {posargs}
```

Comment:
as per the above feedback, its weird to me that you still need to set the env vars for the dockerfiles while also using these flags - could they set marks or something that give dagit everything it needs to figure out which dockerfile to use?